info.go

```diff
@@ -383,6 +411,25 @@ func (pi *ProgramInfo) MapIDs() ([]MapID, bool) {
 	return pi.maps, pi.maps != nil
 }
 
+// LoadTime returns when the program was loaded since boot time.
+//
+// Available from 4.15.
+//
+// The bool return value indicates whether this optional field is available.
+func (pi *ProgramInfo) LoadTime() (time.Duration, bool) {
+	// loadTime and NrMapIds were introduced in the same kernel version.
+	return pi.loadTime, pi.maps != nil
```

Comment:
Maybe just a personal point of view, but I would not mix `maps` and `loadTime` at this point. At the top of my head, I can't think or know a reason, why a valid `loadTime` should be `0`.  
```suggestion
	return pi.loadTime, pi.loadTime > 0
```

Reply:
You're right, `loadTIme` would be 0 if the field isn't available.